course/api/views.py

```diff
@@ -197,18 +201,55 @@ class CourseExercisesViewSet(NestedViewSetMixin,
     parent_lookup_map = {'course_id': 'course_instance.id'}
     serializer_class = CourseModuleSerializer
```

Comment:
If the serializer really isn't used, then it shouldn't be declared in the class at all. But I am curious if this class could still be implemented with the serializer instead of writing all the output manually in the class methods.